fusestore/store/src/meta_service/raft_state.rs

```diff
@@ -44,7 +53,7 @@ impl RaftState {
         config: &configs::Config,
         open: Option<()>,
         create: Option<()>,
```

Comment:
Why not just use boolean type for these two args?

Reply:
In future there may be some create-specific args or open-specific args. 
E.g. 
```
open_create(
    open: Option<Path>, /* open from a path: */
    create: Option<(u64, u64)>, /* create with initial state: */ 
)
```

Reply:
Got it.